strabo/strabo/views/public.py

```diff
@@ -7,24 +7,12 @@
 get_geojson, search, gallery_search
 from strabo.geojson import make_featureCollection
 from strabo.utils import prettify_columns, get_raw_column
+import copy
 
 @app.route("/map")
```

Comment:
Style nit: generally, prefer `'` to `"` for strings in python unless you're capturing quotes. Whichever you go with, pick one and stay consistent.